src/ui/components/accessibility/AccessibilityList.native.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react'
+
+import { render } from 'tests/utils'
+import { AccessibilityList } from 'ui/components/accessibility/AccessibilityList'
+import { Spacer, Typo } from 'ui/theme'
+
+describe('accessibilityList', () => {
+  it.each`
+    itemList                                                                      | numberOfSeparator
+    ${[]}                                                                         | ${0}
+    ${[<Typo.Body key={1}>Item</Typo.Body>]}                                      | ${0}
+    ${[<Typo.Body key={1}>Item</Typo.Body>, <Typo.Body key={2}>Item</Typo.Body>]} | ${1}
+  `(
+    'it should render $itemList.length items and $numberOfSeparator separators',
+    ({ itemList, numberOfSeparator }) => {
+      const Separator = <Spacer.Column numberOfSpaces={6} testID="accessibility-list-separator" />
+
+      const { queryAllByTestId, queryAllByText } = render(
+        <AccessibilityList items={itemList} Separator={Separator} />
+      )
+
+      const separatorList = queryAllByTestId('accessibility-list-separator')
+
+      expect(queryAllByText('Item').length).toEqual(itemList.length)
+      expect(separatorList.length).toEqual(numberOfSeparator)
+    }
+  )
+})
+it('should return 2 items and 1 separator', () => {
+  const tempItemsList: ReactNode[] = [
+    <Typo.Body key={1}>Item 1</Typo.Body>,
+    <Typo.Body key={2}>Item 2</Typo.Body>,
+  ]
+
+  const Separator = <Spacer.Column numberOfSpaces={6} testID="accessibility-list-separator" />
+
+  const { getAllByTestId, getByText } = render(
+    <AccessibilityList items={tempItemsList} Separator={Separator} />
+  )
+
+  const separatorList = getAllByTestId('accessibility-list-separator')
+
+  expect(getByText('Item 1'))
+  expect(getByText('Item 2'))
+  expect(separatorList.length).toEqual(1)
+})
```

Comment:
Est-ce que ce test est vraiment utile ? J'ai l'impression qu'il est redondant avec le it.each du dessus

Reply:
ah bah oui, je voulais le supprimer mais j'ai oublié ^^'